README

```diff
@@ -37,6 +37,8 @@ List of build-time dependencies:
     - `autotools`
   * `ncurses`
 
+When building against the GNU C Library you will need `glibc` v2.14 or later.
```

Comment:
This requirement only affects the Linux htop backend platform BTW - folks can still run pcp-htop on Linux with this glibc (i.e. on RHEL6 and older platforms).  Not sure if/how best to clarify the wording here though.  Perhaps simply:

```
When building htop on Linux with the GNU C Library you will need `glibc` v2.14 or later.  Earlier glibc versions are still supported with pcp-htop on Linux, however.
```

Reply:
Reworded a bit, as I'd prefer to avoid "supported". CentOS 6 is out of support etc.